libcxx/test/std/time/time.syn/formatter.file_time.pass.cpp

```diff
@@ -821,7 +821,9 @@ static void test_valid_values_date_time() {
   // Use the global locale (fr_FR)
   check(
 // https://sourceware.org/bugzilla/show_bug.cgi?id=24054
-#if defined(__GLIBC__) && __GLIBC__ <= 2 && __GLIBC_MINOR__ < 29
+#if defined(__powerpc__) && defined(__linux__)
+      SV("%c='jeu. 01 janv. 1970 00:00:00 UTC'\t%Ec='jeu. 01 janv. 1970 00:00:00 UTC'\n"),
```

Comment:
Is the only difference here the time zone? That seems like the locale on your machine is configured differently.

Reply:
Yes, that is the only difference.  Can we update the test to be more tolerant of different configurations?

Reply:
I think that is quite hard. The C++ code uses libc's locale. Unfortunately different vendors have different view how things are done in a specific locale. I agree this is not great, but most of our locale specific tests have a lot of `#ifdef`s.

Reply:
Actually this clock should always have UTC at timezone, this GMT is wrong too.
[[time.format]/12](http://eel.is/c++draft/time.format#12)
```
Remarks: If %Z is used, it is replaced with STATICALLY-WIDEN<charT>("UTC").
```